dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -142,7 +150,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             co_await winrt::resume_after(std::chrono::seconds(backOffTimeInSeconds));
         }
 
-        
+
```

Comment:
Can we undo the changes in this file. They are all channel related stuff and i dont see anything critical here